src/FinchNotation/syntax.jl

```diff
@@ -43,6 +43,8 @@ const instance_nodes = (
     dimless = :($finch_leaf_instance(dimless))
 )
 
+∂() = 1
+∂(args...) = 0
```

Comment:
do we wanna provide a non-unicode alternative like `d` or `d_`?

Reply:
I'll change `∂` to `d` :)